src/main/java/core/basesyntax/service/StorageServiceImpl.java

```diff
@@ -0,0 +1,34 @@
+package core.basesyntax.service;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.strategy.OperationStrategy;
+import java.util.List;
+
+public class StorageServiceImpl implements StorageService {
+    private List<FruitTransaction> fruitTransactionList;
+    private OperationStrategy transactionHandler;
+
+    public StorageServiceImpl(List<FruitTransaction> fruitTransactionList,
+                              OperationStrategy transactionHandler) {
+        this.fruitTransactionList = fruitTransactionList;
+        this.transactionHandler = transactionHandler;
+    }
+
+    @Override
+    public void transfer() {

```

Comment:
```suggestion
    public void processTransactions() {
```
it's not clear from the method name what are you transferring